server/utils/pluginapi/api.go

```diff
@@ -21,8 +21,28 @@ func New(api plugin.API) *API {
 	}
 }
 
+func (a *API) GetMattermostChannel(mattermostChannelID string) (*model.Channel, error) {
```

Comment:
This seems a bit verbose to call the var `mattermostChannelID`. The reason we do it with `mattermostUserID` is to distinguish between a local user and a remote user ID. But we do not have the concept of a remote channel.

```suggestion
func (a *API) GetMattermostChannel(channelID string) (*model.Channel, error) {
```